gfx/layers/opengl/CompositorOGL.cpp

```diff
@@ -724,8 +724,20 @@ CompositorOGL::BeginFrame(const nsIntRegion& aInvalidRegion,
   // DrawWindowUnderlay. Make sure the bits used here match up with those used
   // in mobile/android/base/gfx/LayerRenderer.java
 #ifndef MOZ_ANDROID_OMTC
-  mGLContext->fClearColor(0.0, 0.0, 0.0, 0.0);
-  mGLContext->fClear(LOCAL_GL_COLOR_BUFFER_BIT | LOCAL_GL_DEPTH_BUFFER_BIT);
+  static int drawsWindowUnderlay = 0;
+  if (drawsWindowUnderlay == 0) {
+    char *windowUnderlay = PR_GetEnv("MOZ_DRAWS_WINDOW_UNDERLAY");
```

Comment:
Are you saying that let's add ClearCompositorContext to the preferences, I cannot see one in gecko 31.

What should be the category? Maybe something like:

```
DECL_GFX_PREF(Once, "gfx.compositor.clear-context", ClearCompoisitorContext, bool, true);
```